### PR TITLE
Navbar Adjustments

### PR DESCRIPTION
**Pull request by seddboi**
- Adjustments to navbar buttons by making them a bit less hard to look at, without sacrificing too much of the color scheme. "navBtns" styling updated, Shop and Subscription buttons remain permanent, Logo size updated, and whole NavBar is mobile responsive.

- Updated Index.css to allow CSS transitions to have cross-browser compatibility.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -3,29 +3,310 @@
 @tailwind utilities;
 
 @layer components {
-  
-.hoverUnderline {
-  display: inline-block;
-  position: relative;
-}
```
Comment (anicmarko9) [inline src/index.css:10]: I've just realized that the person who contributed before you made CSS changes, instead of TailwindCSS. This is his version:
```css
.hoverUnderline {
  display: inline-block;
  position: relative;
}
```
But this is how it was supposed to be, with TailwindCSS:
```css
.hoverUnderline {
  @apply inline-block relative;
}
```
Both of these use cases achieve the same thing, but I prefer this second option, since I'm writing everything in TailwindCSS, and it looks cleaner.

But that was completely my bad, I think that I've missed this part in my previous code review, I'm so sorry.

File: src/index.css [hunk 2]
```diff
@@ -3,29 +3,310 @@
...
+
+  .hoverUnderline::after {
+    @apply bg-primary;
+    content: '';
+    position: absolute;
+    bottom: -4px;
+    left: 0;
+    width: 100%;
+    height: 2px;
+    border-radius: 2px;
```
Comment (anicmarko9) [inline src/index.css:19]: With previous comment being said, can you please transform this `.hoverUnderline` and `.hoverUnderline::after` from CSS to TailwindCSS? You can make another PR, and that will also count as a contribution.
```css
  @apply bg-primary absolute -bottom-1 left-0 w-full h-[2px] rounded-sm <... other options ...>;
```

File: src/index.css [hunk 3]
```diff
@@ -3,29 +3,310 @@
...
+      -webkit-transform 0.35s cubic-bezier(0.465, 0.183, 0.153, 0.946);
+    transition:
+      transform 0.35s cubic-bezier(0.465, 0.183, 0.153, 0.946),
+      -webkit-transform 0.35s cubic-bezier(0.465, 0.183, 0.153, 0.946);
+    transition:
+      transform 0.35s cubic-bezier(0.465, 0.183, 0.153, 0.946),
+      -webkit-transform 0.35s cubic-bezier(0.465, 0.183, 0.153, 0.946);
+    transition:
+      transform 0.35s cubic-bezier(0.465, 0.183, 0.153, 0.946),
+      -webkit-transform 0.35s cubic-bezier(0.465, 0.183, 0.153, 0.946);
```
Comment (anicmarko9) [inline src/index.css:300]:
```css
    transition:
      transform 0.35s cubic-bezier(0.465, 0.183, 0.153, 0.946),
      -webkit-transform 0.35s cubic-bezier(0.465, 0.183, 0.153, 0.946);
```
This code seems like it's repeating over and over again. Is there a reason for that? Can you provide me a link with the documentation that explains this, or just give me your explanation because I don't really understand it.

File: src/index.css [hunk 4]
```diff
@@ -3,29 +3,310 @@
...
   .hoverUnderline:hover::after {
-  transform: scaleX(1);
-  transform-origin: bottom left;
+    -webkit-transform: scaleX(1);
+    -ms-transform: scaleX(1);
+    transform: scaleX(1);
+    -webkit-transform-origin: bottom left;
+    -ms-transform-origin: bottom left;
+    transform-origin: bottom left;
   }
```
Comment (anicmarko9) [inline src/index.css:310]: This also needs to be TailwindCSS. You can put these `scaleX` or other transform options inside of `tailwind.config.js`.

File: src/index.css [hunk 5]
```diff
@@ -37,7 +318,7 @@
...
 
   .navBtns {
-    @apply hover:text-primary hover:bg-white hover:border-primary border-2 border-primary border-opacity-0 transition-all duration-[350ms] ease-in rounded bg-primary px-4 py-2 shadow-lg backdrop-blur-lg backdrop-filter bg-opacity-70 text-white;
+    @apply text-[18px] text-slate-300 border-2 border-slate-200 hover:border-primary transition-all duration-[150ms] ease-in rounded px-2 mr-3 hover:text-primary flex items-center justify-center h-[35px];
   }
```
Comment (anicmarko9) [inline src/index.css:322]: See, you made this `.navBtns` with TailwindCSS, but every other this with regular CSS. It doesn't feel consistent.

**Review by anicmarko9 — APPROVED** [review]
Thank you for your contribution, you can make another PR which will also count as a contribution if you are willing to work further on this issue and clear up some redundant code lines.

**Comment by anicmarko9** [comment]
I've also noticed that you put majority of the changes inside of `./src/index.css`, while there was already existing "styles" directory. I've made another PR and put all your changes into `./src/styles/newStyles.css`, so you can `git pull origin main` and adjust accordingly.

**Comment by seddboi** [comment]
You got it. I'll finish these up ASAP 👍🏼

File: src/index.css (hunk 3 above)
Comment (seddboi) [reply src/index.css:300]: Not sure why this happened, but I removed the repetitions. You are absolutely right about it not being necessary.
